package proto_ff;

import "yd_fieldoptions.proto";


message fashionfashionattributeDesc
{
	optional int32 type = 1[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message fashionfashionActiveAttributeDesc
{
	optional int32 type = 1[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message fashionfashion
{
	optional int64 ID = 1[(yd_fieldoptions.field_cname) = "时装ID"];
	optional string name = 2[(yd_fieldoptions.field_cname) = "时装名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 type = 3[(yd_fieldoptions.field_cname) = "外观类型"];
	optional int32 quality = 4[(yd_fieldoptions.field_cname) = "时装品质"];
	optional string professionLimit = 5[(yd_fieldoptions.field_cname) = "职业限制", (yd_fieldoptions.field_bufsize) = 60];
	optional int64 activationItem = 6[(yd_fieldoptions.field_cname) = "激活道具ID"];
	optional int32 activationNum = 7[(yd_fieldoptions.field_cname) = "激活道具数量"];
	optional int64 starId = 8[(yd_fieldoptions.field_cname) = "升星道具id"];
	optional string starNum = 9[(yd_fieldoptions.field_cname) = "升星道具数量", (yd_fieldoptions.field_bufsize) = 300];
	optional int32 starUp = 10[(yd_fieldoptions.field_cname) = "升星等级上限"];
	optional int32 starBer = 11[(yd_fieldoptions.field_cname) = "每次升星属性增加的百分比值"];
	repeated fashionfashionattributeDesc attribute = 12[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 5];
	repeated fashionfashionActiveAttributeDesc ActiveAttribute = 13[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 6];
}


message Sheet_fashionfashion
{
	repeated fashionfashion fashionfashion_List = 1[(yd_fieldoptions.field_arysize)=20];
}
